coralproject/talk
# Performance enhancements

## What does this PR do?
- [x] Throttled and batched subscription handling
- [x] Preserve immutability and correctly detect when query data has changes (using custom optimzied version of `graphql-anywhere` (https://github.com/cvle/graphql-anywhere/tree/optimized)
- [x] Preoptimize graphql documents
- [x] Decouple apollo redux
- [x] After testing and review: Remove death star button, only optimize documents in production mode

## How do I test this PR?

- Check that everything is still working
- Open moderation panel
- Open stream in another window with another user and click on Death Star button which posts 100 comments in  10 seconds
